pkg/arvo/sys/hoon.hoon

```diff
@@ -16793,7 +16811,27 @@
           ==
         ==
       ==
-    ::
+    ::  parses [a b c] or ==  a  b  c  ==
+    ++  lynx
```

Comment:
Hmm, so this is implemented as a parse-time macro?  Does that mean it doesn't even need an entry in `+hoon`?

We usually add synthetic runes by adding an entry to `+open` instead of the parser.  This allows for better composability -- one synthetic rune can desugar to another.

Reply:
Right. This seems to be mixing parser logic and desugaring logic in one place. Can you please parse this to a %brbs and then desugar that in open:ap?

Reply:
This is an artifact of having translated from +* (where it makes sense that it's a parser macro). The type should have (list term).